ciongke/test-image/Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM ubuntu:16.04
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+    curl \
+    python \
+    gcc \
+    g++ \
+    libc6-dev \
+    make \
+    ca-certificates \
+    && rm -rf /var/lib/apt/lists/*
+
+# gcloud
```

Comment:
Which of these will change the most frequently? If we separate these into so many `RUN` and `ENV` commands, should we order them according to this?